noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/components/reset_output_composer.nr

```diff
@@ -104,24 +93,24 @@ impl<let NH_RR_PENDING: u32, let NH_RR_SETTLED: u32, let NLL_RR_PENDING: u32, le
         nullifiers
     }
 
-    unconstrained fn get_sorted_note_encrypted_log_hashes(
+    unconstrained fn get_sorted_siloed_private_logs(
         self,
-    ) -> [NoteLogHash; MAX_NOTE_ENCRYPTED_LOGS_PER_TX] {
-        let mut log_hashes = sort_by_counter_asc(self.hints.kept_note_encrypted_log_hashes);
-        for i in 0..log_hashes.len() {
-            log_hashes[i].note_hash_counter = 0;
+    ) -> [Scoped<PrivateLogData>; MAX_PRIVATE_LOGS_PER_TX] {
+        let mut private_logs = sort_by_counter_asc(self.hints.kept_private_logs);
+        for i in 0..private_logs.len() {
+            private_logs[i].inner = silo_private_log(private_logs[i]);
+            // The following modifies self.hints.kept_private_logs :(
+            // private_logs[i].inner.log = silo_private_log(private_logs[i]);
```

Comment:
:fearful:

Reply:
Does it also modify the hints if we write directly to private_logs[i].inner.log[0] ?